service.js

```diff
@@ -73,5 +74,38 @@ require('./migrate-psql-db.js')(function (err) {
     .client({type: 'web', port: 10304, pin: {role: 'cd-salesforce', cmd: '*'}})
     .client({type: 'web', port: 10306, pin: 'role:cd-events,cmd:*'});
 });
-
-seneca.act({ role: 'queue', cmd: 'start' });
+seneca.ready(function () {
+  seneca.act({ role: 'queue', cmd: 'start' });
+  var escape = require('./node_modules/seneca-postgresql-store/lib/relational-util.js').escapeStr;
+  ['load', 'list'].forEach(function (cmd) {
+    seneca.wrap('role: entity, cmd: ' + cmd, function filterFields (args, cb) {
+      try {
+        ['limit$', 'skip$'].forEach(function (field) {
+          if (args.q[field] && args.q[field] !== 'NULL' && !(args.q[field] + '').match(/^[0-9]+$/g)) {
```

Comment:
Maybe `!/^[0-9]+$/g.test(args.q[field] + '')` instead since that returns a boolean, rather than an array of matches?